tests/MyGame/Example/Monster.lua

```diff
@@ -359,7 +359,7 @@ function mt:Testarrayofsortedstruct(j)
   local o = self.view:Offset(62)
   if o ~= 0 then
     local x = self.view:Vector(o)
-    x = x + ((j-1) * 4)
```

Comment:
I know this changed because of the changes to `Type::Serialize`, but it was an actual bug in the bfbs implementation.
You can compare the generation to that of the languages that don't use bfbs and you would see it should've been 8 and not 4 because.
The bug is: Struct Vectors are stored as a list of the struct itself while table vectors are stored as a list of offsets.
The current implementation was agnostic to that and gave structs the element_size of an offset field even though their size is their inner struct's size.